man/tpm2_createek.1.md

```diff
@@ -22,7 +22,7 @@ Refer to:
 
   * **-e**, **--endorse-passwd**=_ENDORSE\_PASSWORD_:
     Specifies current endorsement password, defaults to NULL.
-    Passwords should follow the "password formatting standards, see section
+    Passwords should follow the password formatting standards, see section
```

Comment:
"password formatting standards", see section

Reply:
got it, fix incoming